backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/versions/CassandraSchemaVersionDAO.java

```diff
@@ -79,5 +80,9 @@ private PreparedStatement prepareWriteVersionStatement(Session session) {
                 .setUUID(KEY, UUIDs.timeBased())
                 .setInt(VALUE, newVersion.getValue()));
     }
+
+    public Mono<Void> truncateVersion() {
```

Comment:
visible for testing?

Reply:
I think it would be a missuse of the annotation.

IMO @VisibleForTesting is when you increase visibility in order to test that very given method.

Here I use it to test other stuff, thus I think it do not qualify.

Reply:
I'm not sure:
```
/**
 * Annotates a program element that exists, or is more widely visible than otherwise necessary, only
 * for use in test code.
 *
 * @author Johannes Henkel
 */
```
It's likely a production code but only for use in test code, not necessary that method is the testing objective

Reply:
I agree with @trantienduchn : it tags methods you should not use in production code.

Reply:
(I did not see this comment)

Then OK